src/app-layout/split-panel/split-panel-utils.ts

```diff
@@ -30,6 +30,7 @@ function getSplitPanelMinWidth() {
   if (typeof document === 'undefined') {
     return SPLIT_PANEL_MIN_WIDTH;
   }
+  const margin = SPLIT_PANEL_SCROLLBAR_MARGIN / 2;
```

Comment:
I don't quite get why did you declare the margin twice as much and then divided it?

Reply:
The actual margin is 20px which is roughly the width of the scrollbar. However, instead of applying it to a single branch of the ternary below it is added in one case and subtracted in another - this minimises the diff (the breakpoint at which the split panel transition happens).

Reply:
I still don't understand, specifically this part:
> instead of applying it to a single branch of the ternary below it is added in one case and subtracted in another - this minimises the diff

what branch are you referring to?

Reply:
I mean, an alternative could be to have it like that: `return hasScrollbar ? SPLIT_PANEL_MIN_WIDTH - 20 : SPLIT_PANEL_MIN_WIDTH;`